src/helpers/chartOptions.ts

```diff
@@ -7,13 +7,13 @@ export interface LineChartOptions {
 
 export interface BarChartOptions {
     barColor?: ColorDefinition
-    thresholds?: BarChartThreshold[]
+    thresholds?: ChartThreshold[]
 }
 
-export interface BarChartThreshold {
+export interface ChartThreshold {
```

Comment:
It looks like the `ChartThreshold` interface already exists below.

Reply:
I left this as two distinct interfaces for backwards compatibility reasons.  Removing the `BarChartThreshold` and renaming `overThresholdBarColor` is a breaking change that I was avoiding when I created the `TimeSeriesChart`.  It will require code changes in View Builder and requires code changes for anyone who may have written code against this.  I know its annoying, but I'd advise against this unless we think it's absolutely necessary.

Reply:
I reverted this